flight/UAVTalk/uavtalk.c

```diff
@@ -391,7 +377,7 @@ UAVTalkRxState UAVTalkProcessInputStreamQuiet(UAVTalkConnection connectionHandle
 		}
 
 		// Check the lengths match
-		if ((iproc->rxPacketLength + iproc->instanceLength + iproc->timestampLength + iproc->length) != iproc->packet_size) { // packet error - mismatched packet size
+		if ((iproc->rxPacketLength + iproc->instanceLength + iproc->length) != iproc->packet_size) { // packet error - mismatched packet size
```

Comment:
I don't think flight code ever receives timestamped information-- the only place timestamped information is used is in the interpretation of logs.  Removing the ability for flight code to receive timestamped information seems like a good idea.

That is, there's messages flight will no longer accept with this change, but they were messages that were never sent [except to logs] and didn't make sense.  Bumping the version would actually break BC.

Updating the specification to clarify that timestamping is only for logs makes a lot of sense, though.